documentation/asciidoc/computers/compute-module/datasheet.adoc

```diff
@@ -20,17 +20,17 @@ The Raspberry Pi Compute Module 4 (CM 4) is available in a number of different h
 This document describes how to disable various hardware interfaces, in both hardware and software, and how to reduce the amount of memory used by the Linux operating system (OS).
 ****
 
-=== Older Products
+=== Older products
 
-Raspberry Pi CM1, CM3 and CM3L are supported products with an End-of-Life (EOL) date no earlier than January 2026. The Compute Module 3+ offers improved thermal performance, and a wider range of Flash memory options.
+Raspberry Pi CM1, CM3 and CM3L are supported products with an end-of-life (EOL) date no earlier than January 2026. The Compute Module 3+ offers improved thermal performance, and a wider range of flash memory options.
 
 * https://datasheets.raspberrypi.com/cm/cm1-and-cm3-datasheet.pdf[Compute Module 1 and Compute Module 3]
 
-Raspberry Pi CM3+ and CM3+ Lite are supported prodicts with an End-of-Life (EOL) date no earlier than January 2026.
+Raspberry Pi CM3+ and CM3+ Lite are supported products with an end-of-life (EOL) date no earlier than January 2026.
```

Comment:
https://www.raspberrypi.com/products/compute-module-3-plus/ says January 2028.

Reply:
Thanks Andy. Could you please fix it and submit a PR?

Reply:
#3502